modules/calypso-redirects.php

```diff
@@ -0,0 +1,113 @@
+<?php
+
+if ( ! class_exists( 'Calypso_Redirects' ) ) :
```

Comment:
Mainly because I wanted to avoid indenting everything a whole level. The alternative syntax doesn't use brackets, so code editors don't complain about indents. I've seen this same method used other places in the code as well (which is actually where I got the idea from).

- https://github.com/Automattic/jetpack/blob/master/locales.php#L3
- https://github.com/Automattic/jetpack/blob/master/locales.php#L95
-
 https://github.com/Automattic/jetpack/blob/master/modules/shortcodes/presentations.php#L78
- https://github.com/Automattic/jetpack/blob/master/modules/sso/class.jetpack-sso-helpers.php#L3

If you'd like, we can switch this up. We can use brackets and indent the file. Let me know what yo think :)

Reply:
I see, and we can't use a bail here because the Fatal Error would kick in before that. If the whole file is under `if` condition, we might as well move this check to `module-extras.php` when deciding whether to include it. On the other hand, since it seems to be an established pattern, we might keep it as is too. It's up to you. :)